src/F1_plotting.py

```diff
@@ -4,6 +4,9 @@
 import matplotlib.pyplot as plt
 import pandas as pd
 
+import networkx as nx
+import oemof.network.graph as graph
+
```

Comment:
The reason for this being placed in a function was that it can be hard to install `graphviz` - if the import is here, you can not use the whole MVS if you fail to do so.

Reply:
Oh ok I understand, then I think it is more explicit to put the import within a try/except ModuleNotFoundError